README.rst

```diff
@@ -1,132 +1,196 @@
 
-*coala* is written with a lower case "c".
+.. code-block:: bash
 
-Read more at our `documentation <http://coala.rtfd.org/>`__.
+    $ coala --help
 
-To consult our list of Frequently Asked Questions, simply visit `this link <http://coala.readthedocs.io/en/latest/Users/FAQ.html>`__.
+To run coala over your project, create a ``.coafile``, the project-specific
+configuration file that will store all your settings for coala. A sample
+``.coafile`` will look something like this:
 
-Why use coala?
---------------
+.. code-block:: bash
 
-- *coala* provides **built-in checking routines** (named bears in *coala*).
-- **Serves your requirement**: You can easily write your own checks (using
-  bears).
-- *coala* provides **user-friendly interfaces** like json, formatted and
-  interactive output in the CLI and *plugins for various editors* are
-  available as well.
-- **Optimal performance**: *coala* manages parallelizing the checking-routines
-  without you having to worry.
-- **Unified interface**: One tool for all programming languages.
+    [Spacing]
+    files = src/**/*.py
+    bears = SpaceConsistencyBear
+    use_spaces = True
 
-If you want to learn more about *coala*, its functionality and its usage,
-please take a look at our
-`tutorial <http://coala.rtfd.org/en/latest/Users/Tutorials/Tutorial.html>`__.
+* The `files` key tells coala which files to lint - here, we're linting all
+  python files inside the `src` directory.
+* The `bears` key specifies which bears (plugins) to use. We support a huge
+  number of languages and you can find the whole list
+  `here <github.com/coala-analyzer/bear-docs>`__.
+* `use_spaces` enforces spaces over tabs in the codebase. `use_spaces` is a
+  setting for the `SpaceConsistencyBear`.
 
-Installation
-------------
+Store the file in the project's root directory and run coala:
 
-To install *coala* with the official set of analysis routines you can simply run
-``pip3 install coala-bears``.
+.. code-block:: bash
 
-*coala* only, without the official bears, can be installed with
-``pip3 install coala``. If you need more information about the installation and
-dependencies, take a look at our `installation documentation
-<http://coala.rtfd.org/en/latest/Users/Install.html>`__.
+    $ coala
 
-The latest code from master is automatically deployed to PyPI as a
-development version. Get it with ``pip3 install coala --pre``.
+Please read our
+`coafile specification <http://coala.readthedocs.io/en/latest/Users/coafile.html>`__
+to learn more.
 
-|PyPI|
+However, if you don't want to save your settings, you can also run coala with
+command lind arguments:
 
-Authors
--------
+.. code-block:: bash
 
-*coala* is maintained by a growing community. Please take a look at the
-meta information in `setup.py <setup.py>`__ for current maintainers.
+    $ coala --files=setup.py --bears=SpaceConsistencyBear -S use_spaces=True
+
+Note that this command does the same thing as having a coafile and running
+`coala`.
```

Comment:
``The only difference is that having a coafile will save the coafile for other uses.``